src/main/java/com/hubsante/model/rpis/HealthMotive.java

```diff
@@ -62,7 +62,7 @@ public HealthMotive code(String code) {
   }
 
   /**
-   * A valoriser avec un code de la nomenclature CISU-Code_Motif_patient-victime
+   * A valoriser avec un code de la nomenclature CISU-Code_Nature_de_fait
```

Comment:
Pas normal que ça change comme ça pour le RPIS par contre
Je vois qu'un ligne nature de fait / whatsHappen avec codeLabel dans l'onglet RPIS pourtant...
Peut-être faire retourner avec la dernière version du MDD ?

Reply:
En fait autant pour moi, il y a 2 code and label dans RPIS ! J'ai modifié le MDD, ça sera bon quand ça tournera sur auto/model-tracker